src/test/java/com/commercetools/sync/cartdiscounts/utils/CartDiscountUpdateActionUtilsTest.java

```diff
@@ -306,6 +307,25 @@ void buildChangeValueUpdateAction_WithSameAbsoluteValues_ShouldNotBuildUpdateAct
         assertThat(changeValueUpdateAction).isNotPresent();
     }
 
+    @Test
+    void buildChangeValueUpdateAction_WithSameAbsoluteValuesUsingDifferentMonetaryAmountImplementation_ShouldNotBuildUpdateAction() {
```

Comment:
Could you please check the `checkstyleTest` task of the build that causes the build to failed ?

````
> Task :checkstyleTest
[ant:checkstyle] [WARN] /home/travis/build/commercetools/commercetools-sync-java/src/test/java/com/commercetools/sync/cartdiscounts/utils/CartDiscountUpdateActionUtilsTest.java:311: Line is longer than 120 characters (found 133). [LineLength]
````